app/src/main/java/org/simple/clinic/summary/medicalhistory/MedicalHistorySummaryUiRenderer.kt

```diff
@@ -13,9 +13,21 @@ class MedicalHistorySummaryUiRenderer(
   // which will cause tests to break.
   private val medicalHistoryChangedCallback = ValueChangedCallback<MedicalHistory>()
 
+  private val diabetesManagementFlagChangedCallback = ValueChangedCallback<Boolean>()
```

Comment:
I had no idea this class `ValueChangedCallback` existed 😮

Reply:
Yeah, it was added specifically for migrating screens which were using `.distinctUntilChanged` in order to avoid having to update assertions. 😅